src/commands/certs/issue.ts

```diff
@@ -152,85 +152,25 @@ export default async function issue(
-        cert.meta.domain
-      }.`
-    );
-    if (cert.meta.type === 'dns-01') {
-      output.log(
-        `The certificate provider could not resolve the required DNS record queries.`
-      );
-      output.print(
-        '  Read more: https://err.sh/now-cli/cant-solve-challenge\n'
-      );
-    } else {
-      output.log(
-        `The certificate provider could not resolve the HTTP queries for ${
-          cert.meta.domain
-        }.`
-      );
-      output.print(
-        `  The DNS propagation may take a few minutes, please verify your settings:\n\n`
-      );
-      output.print(`  ${dnsTable([['', 'ALIAS', 'alias.zeit.co']])}\n\n`);
-      output.log(
-        `Alternatively, you can solve DNS challenges manually after running:\n`
-      );
-      output.print(
-        `  ${chalk.cyan(`now certs issue --challenge-only ${cns.join(' ')}`)}\n`
-      );
-      output.print(
-        '  Read more: https://err.sh/now-cli/cant-solve-challenge\n\n'
-      );
+  if (cert instanceof ERRORS.CertError) {
+    if (cert.meta.code === 'wildcard_not_allowed') {
+      // Fallback to start cert order when receiving a wildcard_not_allowed error
+      return runStartOrder(output, client, cns, contextName, addStamp, {
+        fallingBack: true
+      });
     }
-    return 1;
-  }
-  if (cert instanceof ERRORS.ConflictingCAARecord) {
-    output.error(cert.message);
-    return 1;
-  }
-  if (cert instanceof ERRORS.TooManyRequests) {
-    output.error(
-      `Too many requests detected for ${cert.meta.api} API. Try again in ${ms(
-        cert.meta.retryAfter * 1000,
-        {
-          long: true
-        }
-      )}.`
-    );
-    return 1;
-  }
-  if (cert instanceof ERRORS.TooManyCertificates) {
-    output.error(
-      `Too many certificates already issued for exact set of domains: ${cert.meta.domains.join(
-        ', '
-      )}`
-    );
-    return 1;
+    return handleCertError(output, cert);
   }
-  if (cert instanceof ERRORS.DomainValidationRunning) {
-    output.error(
-      `There is a validation in course for ${chalk.underline(
-        cert.meta.domain
-      )}. Please wait for it to complete.`
-    );
-    return 1;
-  }
-  if (cert instanceof ERRORS.DomainConfigurationError) {
-    handleDomainConfigurationError(output, cert);
-    return 1;
-  }
-  if (cert instanceof ERRORS.WildcardNotAllowed) {
-    return runStartOrder(output, client, cns, contextName, addStamp, {
-      fallingBack: true
-    });
-  }
-  if (cert instanceof ERRORS.DomainsShouldShareRoot) {
-    output.error(`All given common names should share the same root domain.`);
-    return 1;
+  if (
+    cert instanceof ERRORS.TooManyRequests ||
+    cert instanceof ERRORS.DomainValidationRunning ||
+    cert instanceof ERRORS.CertConfigurationError ||
+    cert instanceof ERRORS.DomainNotFound ||
+    cert instanceof ERRORS.CertsDNSError
+  ) {
+    return handleCertError(output, cert);
```

Comment:
Probably we can make this function return errors unhandled so we can pipe it with other error handlers.